frontend/src/pages/Weather.tsx

```diff
@@ -0,0 +1,232 @@
+                >
+                    <Center p="md">
+                        <Stack spacing="lg">
+                            <TextInput
+                                placeholder="Search location"
+                                onChange={(event) => {
+                                    console.log(event.currentTarget.value);
+                                    setCity(event.currentTarget.value);
+                                }}
+                                value={city}
+                            />
+                            <TextInput
+                                placeholder="Enter days count (1-5)"
+                                onChange={(event) => {
+                                    console.log(event.currentTarget.value);
+                                    setDays(event.currentTarget.value);
+                                }}
+                                value={days}
+                            />
+                            <Button
+                                onClick={() => {
+                                    console.log("clicked");
+                                    console.log(city);
+                                    getWeather();
+                                }}
+                                color="orange"
+                            >
+                                Get Weather
+                            </Button>
+                        </Stack>
+                    </Center>
+                    <Navbar.Section grow> </Navbar.Section>
+                    <Navbar.Section>
+                        <Group
+                            style={{
+                                display: "flex",
+                                justifyContent: "center",
+                                alignItems: "center",
+                            }}
+                        >
+                            <Button variant="default" className={classes.loginButton}>
+                                Log in
+                            </Button>
+                            <Button className={classes.signUpButton} color="orange">
+                                Sign up
+                            </Button>
+                        </Group>
+                    </Navbar.Section>
+                </Navbar>
+            }
+            footer={
+                <Footer height={40} p="md" className={classes.footer}>
+                    GG VK YB ©
+                </Footer>
+            }
+            header={
+                <Header height={{ base: 50, md: 55 }} p="md" className={classes.header}>
+                    <div
+                        style={{
+                            display: "flex",
+                            alignItems: "center",
+                            fontSize: 24,
+                            height: "100%",
+                        }}
+                    >
+                        <MediaQuery largerThan="sm" styles={{ display: "none" }}>
```

Comment:
Same with "styles" here

Reply:
Fixed in 998470d945740fa7df76e28d30496289496c4b60

Reply:
Sorry for the confusion, let's leave `styles={{ display: "none" }}` as it was, because "Burger" is visible when it should not

Reply:
Fixed in 50b871c199781e701f7bcefc157365473f8d3204